src/SmartComponents/BaselinesPage/EditBaseline/helpers.js

```diff
@@ -140,6 +140,42 @@ function makeDeleteSubFactPatch(factToDelete, parentFact, originalAPIBody) {
     return newAPIBody;
 }
 
+function makeDeleteFactsPatch(factsToDelete, originalAPIBody) {
+    let patchBody = [];
+
+    factsToDelete.forEach(function(fact, index) {
+        if (!fact.selected) {
+            if (Array.isArray(fact[2])) {
+                let newFact;
+                let unselectedSubFacts = [];
+
+                fact[2].forEach(function(subFact, idx) {
+                    if (!subFact.selected) {
+                        unselectedSubFacts.push(originalAPIBody.baseline_facts[index].values[idx]);
+                    }
+                });
+
+                newFact = {
+                    name: originalAPIBody.baseline_facts[index].name,
+                    values: unselectedSubFacts
+                };
+
+                patchBody.push(newFact);
+            } else {
+                patchBody.push(originalAPIBody.baseline_facts[index]);
+            }
+        }
+    });
+
+    let patch = makePatchBody(patchBody, originalAPIBody.baseline_facts);
```

Comment:
It can be a bit confusing naming wise to see a patchBody being sent to something that makes a patch body...